### PR TITLE
Adding spa into addons-index

### PR DESCRIPTION
**Pull request by sherrif10**
Added spa module to addons index 
 cc @dkayiwa  @ibacher

### CODE REVIEW

**Comment by sherrif10** [comment]
Am resolving test failure, Am not sure what causes this test to fail because other modules are working fine

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -1558,6 +1558,39 @@
...
+      ],
+      "links":[
+        {
+          "rel":"documentation",
+          "href":"https://wiki.openmrs.org/display/projects/Frontend+Implementer+Documentation",
+          "title":"OpenMRS Wiki"
+        },
+        {
+          "rel":  "source",
+          "href": "https://github.com/openmrs/openmrs-module-spa",
```
Comment (ibacher) [inline src/main/resources/add-ons-to-index.json:1584]:
```suggestion
          "href": "https://github.com/openmrs/openmrs-module-spa"
```

JSON is very strict. You cannot have a comma on the end of a record unless the next thing is a new value.

Comment (sherrif10) [reply src/main/resources/add-ons-to-index.json]: Thanks @ibacher , sure i have resolved that

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -1558,6 +1558,39 @@
...
+          "name":"Burke Mamlin"
+        },
+        {
+          "name":"Brandon Istenes"
+        }
+      ],
+      "links":[
+        {
+          "rel":"documentation",
+          "href":"https://wiki.openmrs.org/display/projects/Frontend+Implementer+Documentation",
```
Comment (dkayiwa) [inline src/main/resources/add-ons-to-index.json:1579]: Is this the project home page?

Comment (sherrif10) [reply src/main/resources/add-ons-to-index.json:1579]: This project does not have specific homepage, currently it is using the same homepage Microfrontends

**Comment by ibacher** [comment]
In light of the [Microfrontend squad's likely decision to abandon this module](https://openmrs.slack.com/archives/CHP5QAE5R/p1600817890001400), I'd suggest we *don't* want to add spa-module to the addons index.

**Comment by sherrif10** [comment]
Thanks for bringing this @ibacher , Actually @brandones suggested that however i haven't got a very clear picture of why this is going this way

**Comment by ibacher** [comment]
Apparently, they're developing a different strategy to integration MFs with OpenMRS that won't rely on having a specific module installed on the OpenMRS platform.

**Comment by sherrif10** [comment]
Thanks for that, does that mean we will definitely abandon this module, cant we add it on addon index for future use, i see there are some modules in openmrs addons that are not much used.

**Comment by brandones** [comment]
@sherrif10 Yes, module-spa is being abandoned. Adding abandoned things to the Addon Index makes the Addon Index less useful. Don't do it.

**Comment by sherrif10** [comment]
Thanks @brandones  for that, i will surely stop it from here as you have said, feel free to comment on the thread on talk

**Pull request closed without merging**